File: CPP/1020 - Idade em dias.cpp
#include <iostream>

using namespace std;

int main() {

	int ano, mes, dia;

	cin >> dia;

	ano = dia/365;
	mes = dia%365/30;
	dia = dia%365%30;

	cout << ano << " ano(s)" << endl;
	cout << mes << " mes(es)" << endl;
	cout << dia << " dia(s)" << endl;

	return 0;
}

File: Go/1020 - Idade em dias.go
package main

import (
	"fmt"
)

func main(){

	var ano, mes, dia int

	fmt.Scan(&dia)

	ano = dia/365
	mes = (dia%365)/30
	dia = (dia%365)%30

	fmt.Printf("%d ano(s)\n", ano)
	fmt.Printf("%d mes(es)\n", mes)
	fmt.Printf("%d dia(s)\n", dia)

}